src/ci_checks/i18n-ai-translate.sh

```diff
@@ -7,7 +7,7 @@ cp i18n/en.json i18n/en-latest.json
 git checkout $merge_base -- i18n/en.json
 if ! diff i18n/en-latest.json i18n/en.json; then
     echo "Updating translations..."
-    npx i18n-ai-translate diff -b i18n/en.json -a i18n/en-latest.json -l "English" --verbose
+    npx i18n-ai-translate diff -b i18n/en.json -a i18n/en-latest.json -l "English" --verbose --engine chatgpt
```

Comment:
does this default to gpt3.5 or 4?

Reply:
Don’t think this is something that should have a default, especially if both are available to users and have different costs

Reply:
Is it something that should be fixed on the library side or something here

Reply:
library, but a result, here as well